src/merge_tools.rs

```diff
@@ -249,7 +255,10 @@ pub fn run_mergetool(
 
     let output_file_contents: Vec<u8> = std::fs::read(paths.get("output").unwrap())?;
     if output_file_contents.is_empty() || output_file_contents == initial_output_content {
-        return Err(ConflictResolveError::EmptyOrUnchanged);
+        return Err(ConflictResolveError::EmptyOrUnchanged {
```

Comment:
Maybe you can instead add `tracing::debug!()` that prints `std::process::Command` struct.

This kind of information is generally useful while debugging tool configuration.

Reply:
I guess I'd have to do that to all the errors then.

What additional information would that show? The one error that happens on a non-zero exit code already shows that.

Also, do you have an example? I'm not entirely clear on where the `tracing::debug!` would go.

Reply:
I think Yuya means that we could add something this just above this line:
```
tracing::debug!("Merge tool didn't change the output. Tool binary: {:?}, args: {args:?},", editor.program);
```

Then the user can rerun the command with `--verbose` to see that output if they're debugging it. I think the idea is that the user would not typically want to see that information since it can also happen with correct configuration, if the user simply decides to not make any changes (right?).

Reply:
Oh, I missed the bit about `std::process::Command`. That's probably a good thing to add to the debug output.

Reply:
Ah, I think I follow now. It sounds like a good idea, but I want to spend the time to check carefully how it works and think about exactly what should go into the normal output and what should only go in the verbose one. I can add a TODO or we can let this PR hang around until I'm in the mood to do that, it's up to you.

I don't think this is at all urgent, I mainly wrote this quickly last night in order to not forget to do it someday.

Reply:
I feel like the verbose output is for debugging, so I think we should rename it to `--debug`. Let me ask others in the Discord chat where more people are likely to see it.

Reply:
I meant something like `let mut cmd = Command::new(); ...; tracing::debug!(?cmd, "spawning merge tool"); cmd.status()`. I'm not familiar with the tracing API, so there might be a better syntax.

The point is that the debug message might be useful even if the command finished successfully.

Reply:
Sounds good, thanks for the suggestion!

> The point is that the debug message might be useful even if the command finished successfully.

Yeah, that's also a good point.

Reply:
I added a commit to do this. It also includes a long TODO. It also happens to undo most of the first commit, but I think that's fine.